src/applications/personalization/profile/tests/e2e/direct-deposit/non-verified-idme.cypress.spec.js

```diff
@@ -0,0 +1,104 @@
+import { PROFILE_PATHS } from '@@profile/constants';
```

Comment:
These tests are probably the most important part of this PR, as they ensure the Direct Deposit section shows the right content if users are not signed in with ID.me or happen to be ID.me but do not have 2FA set up (which apparently isn't even possible for LOA3 ID.me accounts)